src/sentry/integrations/utils/code_mapping.py

```diff
@@ -77,13 +76,20 @@ def filter_source_code_files(files: List[str]) -> List[str]:
 # XXX: Look at sentry.interfaces.stacktrace and maybe use that
 class FrameFilename:
     def __init__(self, frame_file_path: str) -> None:
-        # XXX: Using regexes would be better
-        if not frame_file_path or frame_file_path.find("<") > -1:
-            raise UnsupportedFrameFilename
+        # Using regexes would be better but this is easier to understand
+        if (
+            not frame_file_path
+            or frame_file_path[0] in ["[", "<", "/"]
```

Comment:
Some JS paths start with a backslash and the logic was not working. In the future, we may attempt to solve this case, however, the samples I saw did not show any in_app code.